internal/provider/cached_image_resource.go

```diff
@@ -365,7 +379,11 @@ func (r *CachedImageResource) Create(ctx context.Context, req resource.CreateReq
 		// FIXME: there are legit errors that can crop up here.
 		// We should add a sentinel error in Kaniko for uncached layers, and check
 		// it here.
-		tflog.Info(ctx, "cached image not found", map[string]any{"err": err.Error()})
+		resp.Diagnostics.AddWarning("Cached image not found!", fmt.Sprintf(
```

Comment:
Nit: do we have to so shouty? Is this a common pattern in tf providers?

Reply:
I can remove the bangs!